core/src/main/java/de/jplag/Submission.java

```diff
@@ -79,6 +59,8 @@ public Submission(String name, File submissionRootFile, boolean isNew, Collectio
         this.isNew = isNew;
         this.files = files;
         this.language = language;
+        tokenList = Collections.emptyList();
```

Comment:
I'd instantiate the specific type of list I want to use e.g., ArrayList

Reply:
Should be only a placeholder, as it is replaced later on via the list of tokens provided by the language module. Previously, this was null, but I did not like all the null checks in the code.